src/core/heila-core.js

```diff
@@ -6,6 +6,38 @@ const BPromise = require('bluebird');
 const {knex} = require('../util/database').connect();
 
 
+function checkHeilaTableStatus(user) {
+  return findByUuid(user.uuid)
+    .then(foundHeila => {
+      
+      // if heila table row doesn't exist -> create it
+      if (foundHeila === null && user.heila === true) {
+        console.log('heila not found + heila = true --> add table row');
+        return createHeila(user);
+      // if heila table row exists but it shouldn't -> delete it
+      } else if (foundHeila !== null && user.heila === false) {
+        console.log('heila found + heila = false --> drop table row');
+        return deleteHeila(user);
+      } else {
+        console.log(`heila: ${user.heila} + foundHeila: ${foundHeila} --> doing nothing`);
+        return 1;
```

Comment:
Returning `undefined` would probably be more descriptive.

Reply:
Thanks